packages/cli/tests/integration/rpc.explain.spec.ts

```diff
@@ -3,7 +3,7 @@ import { ExplainRpc } from '@appland/rpc';
 import { Help, explain } from '@appland/navie';
 import { AI } from '@appland/client';
 import { AIClient, AICallbacks, AIInputPromptOptions, AIUserInput } from '@appland/client';
-import { ContextProvider, InteractionHistory, ProjectInfo } from '@appland/navie';
+import { ContextV2, InteractionHistory, ProjectInfo } from '@appland/navie';
```

Comment:
TODO: This would be a good place to ensure that a V1 request is properly handled and a V1 response is returned.